floss/main.py

```diff
@@ -390,42 +391,77 @@ def create_script(sample_file_path, ida_python_file, decoded_strings):
             raise e
     # TODO return, catch exception in main()
 
+KILOBYTE = 1024
+MEGABYTE = 1024 * KILOBYTE
+MAX_FILE_SIZE = 16 * MEGABYTE
```

Comment:
i picked `MAX_FILE_SIZE` out of thin air. we can tweak as appropriate. but i think generating a viv workspace for a 16MB file will take a little bit.